src/metabase/routes.clj

```diff
@@ -31,20 +33,31 @@
 (defn- load-template [path variables]
   (stencil/render-string (load-file-at-path path) variables))
 
+(defn- load-localization []
+  (log/info *locale*)
+  (if (and *locale* (not= (str *locale*) "en"))
+    (try
+      (load-file-at-path (str "frontend_client/app/locales/" *locale* ".json"))
+    (catch Throwable e
+      (log/warn (str "Locale " *locale* " not found."))
```

Comment:
You want this to be `trs`. We're logging that the user's locale couldn't be found, not the system's local.

Reply:
Fixed

Reply:
@senior `(trs *locale*)` does something differently than `(str *locale*)`?

Reply:
@camsaul No, `trs` just calls `(system-locale)` (https://github.com/puppetlabs/clj-i18n/blob/master/src/puppetlabs/i18n/core.clj#L279) and `tru` calls `(user-locale)` (https://github.com/puppetlabs/clj-i18n/blob/master/src/puppetlabs/i18n/core.clj#L269)

Reply:
👍